app/common/elasticsearchclient.py

```diff
@@ -2851,21 +2796,232 @@ def _get_complex_datasource_filter(self, datasources, bol):
             }
         return dict()
 
+
+class AggregationUnitAbstract(AggregationUnit):
+    def build_query_filter(self):
+        if self.filter is not None:
+            self.query_filter = self._get_complex_abstract_kw_filter(self.filter,
+                                                                     BooleanFilterOperator.OR)
+
+    def build_agg(self, filters):
+        self.agg = self._get_abstract_keywords_facet_aggregation(filters)
+
+    def _get_abstract_keywords_facet_aggregation(self, filters):
+        return {
+            "significant_terms": {"field": "private.facets.abstract_lemmas.value"},
```

Comment:
Sorry, didnt get you.  Which field should the cardinality subaggregation use to make the counts ?Did you mean unique_disease_count cardinality sub aggregation (similar to association facets) ? As we are searching for evidence of specific disease-target combination, I dont think unique_target_count makes sense here as it will always be 1. unique_disease_count cardinality subaggregation will count 'child diseases' for that particular abstract term, but I dont see how this is useful at evidence level.